app/client/keybase/doc/vault.md

```diff
@@ -0,0 +1,225 @@
+# Keybase Vault
+
+The Keybase Vault stores key pairs, `(public_key, armoured_private_key)`, using the [Hashicorp Vault](https://www.vaultproject.io/) KV secrets engine. The Keybase Vault is a wrapper around the Hashicorp Vault API.
+
+Keybase Vault requires a few pieces of information to be able to connect to it:
+
+- Address (e.g. `VAULT_ADDR`=http://127.0.0.1:8200/)
+- Token (e.g. `VAULT_TOKEN`=hvs.25YM7qJDN8S2EpFEA4SL0ciD)
+- Mount Path (e.g. /secret)
+
+You can configure the CLI to use a Hashicorp Vault keybase in three different ways, listed in order of precedence:
+
+1. configuration file
+2. environment variables prefixed with POCKET\_ (e.g., POCKET_KEYBASE_TYPE=vault) to override the configuration file
+3. command-line flags (highest precedence)
+
+## Simple Vault Demo
```

Comment:
Note that as a contributor, you can create branches directly in the core repo.

Makes it easier for us to pull and test rather than clone a lot of different repos.

Reply:
oh my bad thought I pushed to the PNI one.

Reply:
ahh there we go should all be sorted out now . <img width="489" alt="Screenshot 2023-03-23 at 23 58 09" src="https://user-images.githubusercontent.com/95193764/227428364-347745a1-112a-4d34-b504-9f7b51178464.png">